spec/lib/rmt/cli/systems_spec.rb

```diff
@@ -133,6 +133,16 @@
           .and change { system.services.count }.from(1).to(0)
           .and change { DeregisteredSystem.count }.by(1)
       end
+
+      context 'when regsharing is set' do
+        it 'de registration is shared with peers' do
+          allow(Settings).to receive(:[]).with(:regsharing).and_return({ peers: 'foo' })
+
+          expect { described_class.start(argv) }
```

Comment:
This test should actually check if `RegistrationSharing.save_for_sharing` gets called correctly.